resilience4j-spring-boot2/src/main/java/io/github/resilience4j/circuitbreaker/monitoring/endpoint/CircuitBreakerHystrixServerSideEvent.java

```diff
@@ -137,5 +117,24 @@ public CircuitBreakerHystrixServerSideEvent(
     private CircuitBreaker getCircuitBreaker(String circuitBreakerName) {
         return circuitBreakerRegistry.circuitBreaker(circuitBreakerName);
     }
+
+    private Flux<ServerSentEvent<String>> getHeartbeatStream() {
+        return Flux.interval(Duration.ofSeconds(1))
+            .map(i -> ServerSentEvent.<String>builder().event("ping").build());
+    }
+
+    private Flux<ServerSentEvent<String>> publishEvents(Seq<Flux<CircuitBreakerEvent>> eventStreams) {
+        BiFunction<CircuitBreakerEvent, CircuitBreaker, String> circuitBreakerEventDataFn = getCircuitBreakerEventStringFunction();
+        return Flux.merge(eventStreams)
+            .onBackpressureDrop()
```

Comment:
`onBackpressureDrop` is needed in case of consumer is not keeping up with producer